backend/climateconnect_api/management/commands/create_test_data.py

```diff
@@ -0,0 +1,106 @@
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+from climateconnect_api.models import (
+    Availability, Role
+)
+from organization.models import (
+    ProjectStatus, Project, Organization
+)
+
+
+def create_availability_test_data(number_of_rows: int):
+    # Based on number of rows you can create test data for availability table.
+    print("Creating user availabilities...")
+    for i in range(number_of_rows):
+        name = "{}-{} hours per week".format(i, i+1)
+        key = name.replace(" ", "")
+        if not Availability.objects.filter(name=name).exists():
+            Availability.objects.create(name=name, key=key)
+        else:
+            print("{} user availability already exists.".format(name))
+
+
+def create_roles_test_data():
+    # Creating 2 roles here
+    print("Creating project roles...")
+
+    if not Role.objects.filter(name='Creator').exists():
+        Role.objects.create(
+            name='Creator', role_type=Role.ALL_TYPE
+        )
+        print("Creator role is successfully created.")
+    else:
+        print("Creator role already exists.")
+    
```

Comment:
I would also add The administrator role in here:
`
if not Role.objects.filter(name='Administrator').exists():
        Role.objects.create(
            name='Administrator', role_type=Role.READ_WRITE_TYPE
        )
        print("Administrator role is successfully created.")
    else:
        print("Administrator role already exists.")
`

I think this makes sense because there is only 3 roles on the live server anyways and this isn't very likely to change in the near future.